packages/@aws-cdk/aws-batch/lib/compute-environment.ts

```diff
@@ -18,6 +18,16 @@ export enum ComputeResourceType {
    * Resources will be EC2 SpotFleet resources.
    */
   SPOT = 'SPOT',
+
+  /**
+   * Resources will be Fargate resources.
+   */
+  FARGATE = 'FARGATE',
+
+  /**
+   * Resources will be Fargate resources.
+   */
+  FARGATE_SPOT = 'FARGATE_SPOT',
```

Comment:
What does *spot* mean in the context of *fargate*? would be nice to have a sentence or two about it.